index.html

```diff
@@ -316,7 +316,17 @@
                     // toggles "Loading..."
                     loading: false
                 },
-                created: function() { this.refreshBuilds() },
+                created: function() {
+                    const searchParams = new URLSearchParams(window.location.search);
+                    if (searchParams.has('stream')) {
+                        const queryStream = searchParams.get('stream');
+                        if (["stable", "testing", "next", "testing-devel", "next-devel", "bodhi-updates", "developer"].includes(queryStream)) {
```

Comment:
Hmm, this is unfortunate. There should be a cleaner way to do this where we only have one source of truth array variable containing the streams that we can also use to populate the dropdown options higher up. Anyway, not a blocker!

Reply:
Yes, I was thinking about adding a source of truth and using that instead.
Will follow up with a PR after tested locally.